docs/getting-started/ios/fastlane-swift.md

```diff
@@ -0,0 +1,39 @@
+# Getting Started with Fastlane.swift
+
+Welcome to Fastlane.swift. Fastlane.swift allows you to write your _fastlane_ configuration using Xcode, in Swift - the language you have come to know and love from the world of iOS development..
+
+## Currently Supported:
+
+Fastlane.swift currently supports all actions in the main, [_fastlane_](https://github.com/fastlane/fastlane) repository. The API for each action is automatically generated and will accept all options and parameters.
+
+## Get Started:
+
+### Step 1:
+
+Run the following command in your terminal:
+
+``` no-highlight
+fastlane init swift
+```
+
+### Step 2:
+
+Open the file located at `<path/to/your/project>/fastlane/swift/FastlaneRunner/FastlaneRunner.xcodeproj` to configure `Fastfile.swift` to your liking.
+
+### Step 3:
+
+Run `fastlane <laneName>` in your terminal to execute _fastlane_.
```

Comment:
I think the convention we have is `lane_name` 👍

Reply:
Not in Swift though. :/

Reply:
Ah right 👍